ontology/t4fs.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: ontology_detail
+id: t4fs
+title: terms4FAIRskills
+contact:
+  email: pete_mcq@protonmail.com
+  github: drosophilic
+  label: Peter McQuilton
+  orcid: 0000-0003-2687-1982
+description: A terminology for the skills necessary to make data FAIR and to keep it FAIR.
+domain: information
+homepage: https://github.com/terms4fairskills/FAIRterminology
+license:
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0/
+preferredPrefix: TF4S
+products:
+- id: t4fs.owl
+- id: t4fs.obo
+- id: t4fs.json
+- id: t4fs-community.owl
```

Comment:
can you add "title" keys for all of these so we can tell what the difference is between the base ones and the `-community` ones?